### PR TITLE
qdrant: Documentation for the new QdrantVectorStore class

### PR DESCRIPTION
**Pull request by Anush008**
## Description

Follow up on #24165. Adds a page to document the latest usage of the new `QdrantVectorStore` class.

### CODE REVIEW

File: docs/docs/how_to/hybrid.ipynb
Comment (efriis) [inline docs/docs/how_to/hybrid.ipynb:1]: please don't specifically promote qdrant over the other ones here. feel free to add this as a note in the qdrant page.

File: docs/docs/integrations/platforms/index.mdx
Comment (efriis) [inline docs/docs/integrations/platforms/index.mdx:1]: it's one provider. please merge these two entries

File: docs/docs/integrations/platforms/index.mdx [hunk 1]
```diff
@@ -38,6 +38,7 @@ These providers have standalone `langchain-{provider}` packages for improved ver
 - [OpenAI](/docs/integrations/platforms/openai)
 - [Pinecone](/docs/integrations/providers/pinecone)
 - [Qdrant](/docs/integrations/providers/qdrant)
+- [Qdrant - NEW](/docs/integrations/providers/qdrant-new)
```
Comment (efriis) [inline docs/docs/integrations/platforms/index.mdx:41]: broken link? qdrant-new was a vectorstore not a provider. also please merge into one entry

File: docs/docs/integrations/vectorstores/qdrant-new.ipynb
Comment (efriis) [inline docs/docs/integrations/vectorstores/qdrant-new.ipynb:1]: merge into existing entry. remove the deprecated one or mark it deprecated at the bottom of the doc.

File: docs/docs/integrations/platforms/index.mdx (hunk 1 above)
Comment (Anush008) [reply docs/docs/integrations/platforms/index.mdx]: [`67cb5aa` (#24166)](https://github.com/langchain-ai/langchain/pull/24166/commits/67cb5aad6ad70ae1eb44aa4059b68633eee83abf)

File: docs/docs/how_to/hybrid.ipynb
Comment (Anush008) [reply docs/docs/how_to/hybrid.ipynb:1]: Done. [`0a6f0e5` (#24166)](https://github.com/langchain-ai/langchain/pull/24166/commits/0a6f0e5f602bc69f4cb1c2abd87d036773fdce2f)

File: docs/docs/integrations/platforms/index.mdx (hunk 1 above)
Comment (Anush008) [reply docs/docs/integrations/platforms/index.mdx]: I had added this entry before creating the page. I incorrectly predicted the location.
Thanks.

**Review by efriis — COMMENTED** [review]
hey! I think my comment wasn't clear. Let's delete `qdrant-new.ipynb` and merge any relevant contents into qdrant.ipynb! If the old one is deprecated/not recommended for new users, let's either delete those sections of qdrant.ipynb, or put them at the bottom with a note that you recommend using the new implementation instead.

As we've discussed before, having these two separate implementations is going to be confusing for users, so some notes about why someone might use one vs the other would be appreciated as well!

**Comment by Anush008** [comment]
Hey @efriis. I've updated the changes into `qdrant.ipynb` and deleted the new page.

**Review by efriis — APPROVED** [review]
looks great thank you!

**Pull request merged**